docker/api/image.py

```diff
@@ -271,8 +271,8 @@ def load_image(self, data):
         Args:
             data (binary): Image data to be loaded.
         """
-        res = self._post(self._url("/images/load"), data=data)
-        self._raise_for_status(res)
+        return self._result(
```

Comment:
We may not want to do this for API < 1.24 depending on what the content is befor engine 1.12.